_data/events_gdcr2018/Cologne.json

```diff
@@ -3,14 +3,15 @@
   "url": "https://www.meetup.com/de-DE/Softwerkskammer-Koln/events/255294577/",
   "moderators": [
     "@skorzinetzki",
-    "@bikratz"
+    "@bikratz",
+	  "Martin Dieblich"
```

Comment:
Could we align this line properly? I see some extra (and not needed) whitespaces here.

Reply:
of course. i think this came from merging using the github web gui.

Reply:
sorry, that was too fast, now it is fixed... ;)